ui/src/components/shared/SiderLayoutContent.tsx

```diff
@@ -62,36 +64,35 @@ export function SiderLayoutContent({
   }, [warningMessage, isShowingWarning, warningKey]);
 
   return (
-    <>
-      <Layout.Header
-        className="bg-light"
-        style={{ height: "auto", padding: 32, paddingBottom: showDivider ? 0 : 12 }}
+    <Layout className="bg-light" style={{ minHeight: "100vh" }}>
+      <Row
+        gutter={[0, 16]}
+        justify="space-between"
+        style={{ height: "auto", padding: md ? 32 : 16, paddingBottom: showDivider ? 0 : 12 }}
```

Comment:
I see this in the console

<img width="1064" alt="image" src="https://github.com/0xPolygonID/issuer-node/assets/15896177/09a1e5d4-4fe5-48fc-8cec-f90e9d207edc">